### PR TITLE
Move chrony to platform cookbook and transform it into resource

### PR DESCRIPTION
**Pull request by awslitvit**
### Description of changes
Move `chrony` to platform cookbook and transform it into resource

### Tests
* `./kitchen.docker.sh platform-install verify chrony -c 6`
* `./kitchen.docker.sh platform-config verify chrony -c 6`
* `./kitchen.ec2.sh platform-install verify chrony -c 6`
* `./kitchen.ec2.sh platform-config verify chrony -c 6`
* alinux2 AMI successfully built on this code

### Checklist
- [x] Make sure you are pointing to **the right branch** and add a label in the PR title (i.e. **2.x** vs **3.x**)
- [x] Check all commits' messages are clear, describing what and why vs how.
- [x] Make sure **to have added unit tests or integration tests** to cover the new/modified code.
- [x] Check if documentation is impacted by this change.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: cookbooks/aws-parallelcluster-platform/kitchen.platform-config.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    - test
+
+suites:
+  - name: chrony
+    run_list:
+      - recipe[aws-parallelcluster-tests::setup]
+      - recipe[aws-parallelcluster-tests::test_resource]
+    verifier:
+      controls:
+        - /tag:install_chrony/
```
Comment (eantonin) [inline cookbooks/aws-parallelcluster-platform/kitchen.platform-config.yml:14]: question: Shouldn't this be `tag:config_chrony_service_configured` or `tag:config_chrony`

File: cookbooks/aws-parallelcluster-platform/test/controls/chrony_spec.rb [hunk 2]
```diff
@@ -9,7 +9,7 @@
 # This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or implied.
 # See the License for the specific language governing permissions and limitations under the License.
 
-control 'chrony_installed_and_configured' do
+control 'tag:install_chrony' do
   title 'Test chrony installation and configuration'
 
   only_if { !os_properties.redhat_ubi? }
```
Comment (eantonin) [inline cookbooks/aws-parallelcluster-platform/test/controls/chrony_spec.rb:15]: question, non-blocking: Is there a way to test the installation of `npt*` packages?

File: .github/workflows/dokken-unit-tests.yml [hunk 3]
```diff
@@ -122,7 +122,7 @@ jobs:
           - ephemeral-drives-mounted
           - ephemeral-drives-skipped
           - sudo-configured
-          - /chrony-configured/
```
Comment (eantonin) [inline .github/workflows/dokken-unit-tests.yml:125]: Question, non-blocking: Just for the sake of curiosity, how this worked before this PR, I don't see any test respecting `chrony-configured` regex

File: cookbooks/aws-parallelcluster-platform/kitchen.platform-config.yml (hunk 1 above)
Comment (eantonin) [reply cookbooks/aws-parallelcluster-platform/kitchen.platform-config.yml:14]: Agreed offline this will be addressed in next PR

**Review by eantonin — APPROVED** [review]
Approving as comments will be addressed in next PR

File: .github/workflows/dokken-unit-tests.yml (hunk 3 above)
Comment (awslitvit) [reply .github/workflows/dokken-unit-tests.yml:125]: I guess it didn't...

File: cookbooks/aws-parallelcluster-platform/test/controls/chrony_spec.rb (hunk 2 above)
Comment (awslitvit) [reply cookbooks/aws-parallelcluster-platform/test/controls/chrony_spec.rb:15]: We can take a look later.

**Pull request merged**